pkgs/top-level/all-packages.nix

```diff
@@ -17287,7 +17287,9 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) SystemConfiguration CoreFoundation Security;
   };
 
-  squeak = callPackage ../development/compilers/squeak { };
+  squeak = callPackage ../development/compilers/squeak {
+    stdenv = clangStdenv;
```

Comment:
Does this need to be `clangStdenv` other than for testing?

Reply:
When omitted, `nix-build -A squeak` errors as follows:

```
Configuring Squeak 5.0-202003021730 (5.3-19459-64bit) for x86_64-pc-linux-gnu

checking whether make sets $(MAKE)... yes
checking for gcc... clang
checking whether the C compiler works... no
configure: error: in `/build/source/build.linux64x64/squeak.cog.spur/build':
configure: error: C compiler cannot create executables
See `config.log' for more details
```

I don't know why, in the default stdenv, it detects clang, as it's no longer passed into the environment. If I pass in `gccStdenv`, `clang` is still detected.

So, yes, I think this does have to be `clangStdenv`.